controls/ccn_rhel9.yml

```
@@ -0,0 +1,836 @@
+      - audit_rules_dac_modification_chown
+      - audit_rules_dac_modification_fchmod
+      - audit_rules_dac_modification_fchmodat
+      - audit_rules_dac_modification_fchown
+      - audit_rules_dac_modification_fchownat
+      - audit_rules_dac_modification_fremovexattr
+      - audit_rules_dac_modification_fsetxattr
+      - audit_rules_dac_modification_lchown
+      - audit_rules_dac_modification_lremovexattr
+      - audit_rules_dac_modification_lsetxattr
+      - audit_rules_dac_modification_removexattr
+      - audit_rules_dac_modification_setxattr
+
+  - id: A.3.SEC-RHEL8
+    title: Changes to cron settings and scheduled tasks including startup scripts are audited
+    original_title: Se auditan los cambios en la configuración de Cron y en tareas programadas incluyendo los de scripts de inicio.
+    levels:
+      - advanced
+    status: pending
+    notes: |-
+      Some possible rules were included here but it is not clear if the requirement intends to
+      check more than these rules. We can se if more related rules are available in the project
+      and include everything that makes sense in the context of cron and chrony.
+    related_rules:
+      - audit_rules_time_adjtimex
+      - audit_rules_time_settimeofday
+      - audit_rules_time_clock_settime
+      - audit_rules_time_stime
+      - audit_rules_time_watch_localtime
+
+  - id: A.3.SEC-RHEL9
+    title: Attempts to access critical items are audited
+    original_title: Se auditan los intentos de acceso a elementos críticos.
+    levels:
+      - advanced
+    status: planned
+    related_rules:
+      - audit_rules_unsuccessful_file_modification_creat
+      - audit_rules_unsuccessful_file_modification_ftruncate
+      - audit_rules_unsuccessful_file_modification_open
+      - audit_rules_unsuccessful_file_modification_openat
+      - audit_rules_unsuccessful_file_modification_truncate
+
+  - id: A.3.SEC-RHEL10
+    title: All mount operations on the system and changes to the swap are audited
+    original_title: Se audita toda operación de montaje en el sistema y modificaciones en la memoria de intercambio.
+    levels:
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      We probably have audit related rule to monitor mount related syscalls, but it is not clear
+      about the swap. Is the intention to monitor when swap is changed?
+    related_rules:
+      - audit_rules_media_export
+
+  - id: A.3.SEC-RHEL11
+    title: Modifications in PAM files are audited
+    original_title: Se auditan modificaciones en ficheros PAM.
+    levels:
+      - advanced
+    status: pending
+    notes: |-
+      The intention here is probably to audit changes in /etc/pam.d files, but we need to confirm
+      this assumption and get more context.
+
+  #[A.4] Manipulación de los ficheros de configuración
+  - id: A.4.SEC-RHEL1
+    title: Common users do not have local administrator permissions or are included in a sudo group
```

```suggestion
    title: Common users do not have local administrator permissions and are not included in a sudo group
```
This phrasing may be more clear.